Api/V8/Exception/BeanNotFoundException.php

```diff
@@ -0,0 +1,8 @@
+<?php
+namespace Api\V8\Exception;
+
+class BeanNotFoundException extends NotFoundException
+{
+    const CODE_BY_ID = 1;
```

Comment:
These codes should be in the ExceptionCode class.

Reply:
As I mentioned above, I've just made a working example. Let's talk about a possible merge between this and yours.